spec.html

```diff
@@ -42664,14 +43085,45 @@ <h1>SharedArrayBuffer.prototype.constructor</h1>
         <p>The initial value of `SharedArrayBuffer.prototype.constructor` is %SharedArrayBuffer%.</p>
       </emu-clause>
 
+      <emu-clause id="sec-sharedarraybuffer.prototype.grow">
+        <h1>SharedArrayBuffer.prototype.grow ( _newLength_ )</h1>
+        <p>This method performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferMaxByteLength]]).
+          1. If IsSharedArrayBuffer(_O_) is *false*, throw a *TypeError* exception.
+          1. Let _newByteLength_ be ? ToIntegerOrInfinity(_newLength_).
```

Comment:
```suggestion
          1. Let _newByteLength_ be ? ToIndex(_newLength_).
```

Same `ToIndex` suggestion applies here.

Reply:
Agreed, `ToIndex` is better here. I'm going to say this is editorial. I suppose it's technically observable from the perspective of the host hook, but since the hook already says it must throw if the new length is < current length, and a negative length will always be < current length, it's a non-normative change from the perspective of programs.